specification/hdinsight/hdinsight-kafka-rest-proxy/proxy.json

```diff
@@ -450,11 +450,11 @@
       "required": [ "value" ],
       "properties": {
         "value": {
-          "type": "object",
+          "type": "string",
```

Comment:
hello, is this management plan or data plan? is this a breaking change?

Reply:
It supports type: string and type:int  which is why we put it as object.
Currently no customer is using this right now, so we are ok with updating the documentation with this change.

Should I use a type:string/int  ?Is it possible. If not, I prefer it to be updated to string, even though its breaking change.

Reply:
HI @anushreeringne , SDKs are generated based on Swagger. If Swagger changed with breaking change in same version, SDK will contain breaking change also. That's the concern. I don't think type could be `string/int`. Suggest to do it in a new version. definition of breaking change: https://github.com/Azure/adx-documentation-pr/wiki/Breaking-changes-guidelines#breaking-changes